src/lib/containers/table/SynapseTable.tsx

```diff
@@ -523,10 +529,11 @@ export default class SynapseTable extends React.Component<
     const isShowingAccessColumn: boolean | undefined =
       showAccessColumn && this.state.isEntityView
     const isLoggedIn = !!this.context.accessToken
+    const containsGroupBy = isGroupBy(lastQueryRequest.query.sql)
     const isShowingAddToV2DownloadListColumn: boolean =
-      this.state.isFileView && !this.props.hideDownload && isLoggedIn
+      this.state.isFileView && !this.props.hideDownload && isLoggedIn && !containsGroupBy
```

Comment:
I don't think `isFileView` will work for Datasets as they don't have a viewTypeMask (yet)